en.po

```diff
@@ -166,10 +166,10 @@ msgstr "Paid by Local Ministry/Entity"
 
 # https://netsuite.helpscoutdocs.com/article/1088-hr-department-mcc
 msgid "hrDepartment"
-msgstr "HR Department"
+msgstr "Strategy / Capacity (HR Department in Netsuite)"
 
-msgid "status"
-msgstr "Status"
+msgid "organizationalStatus"
+msgstr "Organizational Status"
```

Comment:
We have to be careful about changing the `msgid` of any label. This means that the old label will become invalid in **all** the languages. Right now we only have 3 languages, so it's not too hard to update them all. But in general, we should try to avoid changing the `msgid`.

In this case, "status" is a very generic word. And maybe in the future, there will be a new label for "status" that is different from "organization status". So it does make sense to change the `msgid` here. This part is approved.

However, you should also update the `ar.po` file. You don't need to change the actual translation, just update the `msgid` in there. The translators on All Tongues will be able to change the translation if needed.

Reply:
Okay, Thanks!